include/nil/blueprint/bls_signature/bls12_381_pairing.hpp

```diff
@@ -48,6 +48,7 @@ namespace nil {
             circuit_proxy<crypto3::zk::snark::plonk_constraint_system<BlueprintFieldType, ArithmetizationParams>> &bp,
             assignment_proxy<crypto3::zk::snark::plonk_constraint_system<BlueprintFieldType, ArithmetizationParams>>
                 &assignment,
+                const std::vector<typename BlueprintFieldType::value_type> &internal_storage,
```

Comment:
Why can't we put a pointer to the storage into `common_component_parameter`? Is'n its purpose to easily extend component signature without changing hundred lines of code?